### PR TITLE
Ysn branch

### PR DESCRIPTION
**Pull request by ysnblgn**
ysn branch is done

### CODE REVIEW

**Review by esalkan — COMMENTED** [review]
Hello Yasin, Welcome to fun house ;)

**Review by esalkan — COMMENTED** [review]
Great, you started to work. Go on please...

File: ysn.txt [hunk 1]
```diff
@@ -1 +1,2 @@
-//work1
\ No newline at end of file
+//work1
```
Comment (esalkan) [inline ysn.txt:1]: Write here some java codes. what about "Hello World!" ;)

**Review by esalkan — CHANGES_REQUESTED** [review]
Write here some "Hello World" java code.

File: ysn.txt (hunk 1 above)
Comment (ysnblgn) [reply ysn.txt:1]: ok I'm writing

**Review by esalkan — APPROVED** [review]
@ysnblgn  all is well mate. I am going to merge that changes. good job.

**Comment by ysnblgn** [comment]
> @ysnblgn all is well mate. I am going to merge that changes. good job.

thank you master

**Pull request merged**